.github/CODEOWNERS

```diff
@@ -4,6 +4,7 @@
 /internal/resources/cloud/*                 @grafana/platform-monitoring @grafana/grafana-com-maintainers
 /internal/resources/cloudprovider/*         @grafana/platform-monitoring @grafana/middleware-apps
 /internal/resources/connections/*           @grafana/platform-monitoring @grafana/middleware-apps
+/internal/resources/k6/*                    @grafana/platform-monitoring @grafana/k6-core
```

Comment:
Looks like I don't have permission to add this team to the repository, can you help, please? 🙏🏻  cc/ @spinillos @Duologic

Reply:
Could you also add: `grafana/k6-backend`, please?
This will likely be also co-owned by https://github.com/orgs/grafana/teams/k6-backend